ImperatorToCK3/Source/CK3/Titles/Title.cpp

```diff
@@ -160,7 +166,24 @@ void CK3::Title::initializeFromTag(std::shared_ptr<Imperator::Country> theCountr
 	
 	// --------------- Adjective Locs
 	trySetAdjectiveLoc(localizationMapper);
+}
+
+void CK3::Title::updateFromTitle(const std::shared_ptr<Title>& otherTitle)
+{
+	if (titleName != otherTitle->titleName)
+		throw std::runtime_error(titleName + " can not be updated from  " + otherTitle->titleName + ": different title names!");
```

Comment:
(often best to throw a warning and return, stuff happens, inputs are iffy)

Reply:
Replaced with error level log.